libs/@wingcloud/framework/src/main.ts

```diff
@@ -76,6 +77,9 @@ This is a Wing app and must be run through the Wing CLI (npm install -f winglang
         new std.Test(this, name, handler, props);
       });
     }
+    get _liftMap(): LiftDepsMatrixRaw {
```

Comment:
Would it make sense to make `_liftMap` optional and default to `{}`?

There were a bunch of times (can't recall the exact scenario) where I ran into an issue because this property was not implemented.